feature_tests/dart/lib/src/lib.g.dart

```diff
@@ -176,11 +229,24 @@ final class _ResultOpaqueErrorStructFfi extends ffi.Struct {
 
   @ffi.Bool()
   external bool isOk;
+
+  
+  factory _ResultOpaqueErrorStructFfi.ok(ffi.Pointer<ffi.Opaque> val) {
+    final struct = ffi.Struct.create<_ResultOpaqueErrorStructFfi>();
+    struct.isOk = true;
+    struct.union.ok = val;
+    return struct;
+  }
+  factory _ResultOpaqueErrorStructFfi.err(_ErrorStructFfi val) {
+    final struct = ffi.Struct.create<_ResultOpaqueErrorStructFfi>();
+    struct.isOk = false;
+    struct.union.err = val;
+    return struct;
+  }
 }
 
 final class _ResultOpaqueInt32Union extends ffi.Union {
   external ffi.Pointer<ffi.Opaque> ok;
-
```

Comment:
...

Reply:
this should be fine? I'd rather have that whitespace be adjacent anyway, instead of complicating the template with a manual `if is_ok() & is_err()` subst _just for whitespace_.

Reply:
The template should add an empty line after each field, there shouldn't need to be any special casing.

Reply:
I can't see a way to do this without either:

 - dedenting some of the `{%`s (because removing the `-` also reintroduces the indentation)
 - adding some kind of `if` somewhere

Reply:
I pushed the dedent solution. It works, I'm not happy with it